### ISSUE TITLE
[BUG] improve accessibility with script supporting elements in unordered list

### ISSUE THREAD

**Issue by Anmol-Baranwal** [rupali-codes/LinksHub#1464]
### Description

- As shown in the image
- improve accessibility by using `<li>` under `<ol>`

### Screenshots

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/61694cef-ab86-4c17-b2c8-ae9c2141ad16)

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

**Comment by CBID2** [comment]
Hey @Anmol-Baranwal, are you working on this?

**Comment by CBID2** [comment]
@Anmol-Baranwal, how are you doing with this?

**Comment by Anmol-Baranwal** [comment]
Tried this, but was unable to do. If you want to make this open, feel free to do so.

I'm already very busy building UIVerse.

**Comment by CBID2** [comment]
> Tried this, but was unable to do. If you want to make this open, feel free to do so.
> 
> 
> 
> I'm already very busy building UIVerse.

Thanks for trying
